packages/select/src/components/select/select.tsx

```diff
@@ -129,7 +135,18 @@ export class Select<T> extends AbstractPureComponent2<SelectProps<T>, ISelectSta
 
     private renderQueryList = (listProps: IQueryListRendererProps<T>) => {
         // not using defaultProps cuz they're hard to type with generics (can't use <T> on static members)
-        const { filterable = true, disabled = false, inputProps = {}, popoverProps = {} } = this.props;
+        const { fill, filterable = true, disabled = false, inputProps = {}, popoverProps = {} } = this.props;
+
+        const childProps: { fill?: boolean } = {};
+
+        if (fill) {
+            popoverProps.fill = true;
+            childProps.fill = true;
+        }
+
+        const children = React.Children.map(this.props.children, (child: React.ReactNode) => {
```

Comment:
I don't love this approach and would prefer not to clone children if we can avoid it. We can't guarantee that the children will support the `fill` prop, and I'd rather not send along extraneous HTML attributes which might get rendered out to the DOM (and produce a React warning). 

If we stick to just rendering `{this.props.children}`, then `<Select>` users will have to add `fill` to children themselves (or style it some other way), but I think that's ok. We should make a note of that in the fill prop documentation.